File: Set-21-2021/C/ex001.c
/*
    Esse programa é feito para calcular a soma de n alunos
    E a média das idades
*/


#include <stdio.h>

int main()
{
    int alunos, idade, soma, i;
    float media;
    printf("Digite a quantidade de alunos: ");
    scanf("%d", &alunos);
    
    soma = 0;
    
    for (i=0; i<alunos; i++ )
    {
        printf("Digite a idade do %d° aluno: ", i + 1);
        scanf("%d", &idade);
        soma = soma + idade;
    }
    media = soma / alunos;
    printf("A soma das idades dos %d alunos é %d", alunos, soma);
    printf("\nA média das idades é %.1f", media);
}

File: C C++/Set-21-2021/Aula/Python/ex001.py
# Programa para efetuar soma de idade de n alunos
# E a média das idades

nAlunos = int(input("Qual o número de alunos? "))
i = 0
soma = 0
while i < nAlunos:
    idade = int(input("Digite a idade do {}° aluno: ".format(i+1)))
    soma = soma + idade
    i = i + 1
print("\nA soma das idades é: {}".format(soma))
print("A média das idades é: {}".format(soma/nAlunos))
